src/components/ContactUsForm.jsx

```diff
@@ -0,0 +1,129 @@
+import { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import TextInput from "./TextInput";
+import MultiSelectCheckbox from "./MultiSelectCheckbox";
+import Button from "./Button";
+import postSignUp from "../api/post-signup";
+import SuccessfulCard from "./SuccessfulCard";
+import LoadingSpinner from "./LoadingSpinner";
+
+const emailPattern = "[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+.[a-zA-Z]{2,}$";
+
+
+export default function ContactUsForm() {
+  const [formState, setFormState] = useState("");
+  const [credentials, setCredentials] = useState({
+    name: "",
+    email: "",
+    subject: "",
+   
+  });
+  const [error, setError] = useState({
+    field: "",
+    errorMessage: "",
+  });
+  const handleChange = (e) => {
+    const { id, value } = e.target;
+    setCredentials((prev) => ({
+      ...prev,
+      [id]: value,
+    }));
+  };
+
+  const validateEmail = (email) => {
+    const emailPattern = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
+    return emailPattern.test(email);
+  };
+  const handleSubmit = (e) => {
+    e.preventDefault();
+        
+  };
+  return (
+    <div className="form-container pt-6">
+      {formState === "successful" ? (
+        <SuccessfulCard>
+          <p className="text-lg">Sign up was successful!</p>
+          <Button
+            buttonType="link"
+            href="/login"
+            size="md"
+            buttonStyle="secondary"
+          >
+            Login
+          </Button>
+        </SuccessfulCard>
+      ) : formState === "error" ? (
+        // Todo: handle exceptions
+        <p>Error while submitting the sign up form</p>
```

Comment:
Please change the successful message, something like "Thanks for contacting us, we'll get back to you shortly."
And also button should navigate to the home page, with Go back to home text.